src/checkoutservice/main.go

```diff
@@ -210,19 +210,19 @@ func (cs *checkoutService) Watch(req *healthpb.HealthCheckRequest, ws healthpb.H
 }
 
 func (cs *checkoutService) PlaceOrder(ctx context.Context, req *pb.PlaceOrderRequest) (*pb.PlaceOrderResponse, error) {
-	span := trace.SpanFromContext(ctx)
-	span.SetAttributes(
-		attribute.String("app.user.id", req.UserId),
-		attribute.String("app.user.currency", req.UserCurrency),
-	)
-	log.Infof("[PlaceOrder] user_id=%q user_currency=%q", req.UserId, req.UserCurrency)
-
-	var err error
-	defer func() {
-		if err != nil {
-			span.AddEvent("error", trace.WithAttributes(attribute.String("exception.message", err.Error())))
-		}
-	}()
+	// span := trace.SpanFromContext(ctx)
+	// span.SetAttributes(
+	// 	attribute.String("app.user.id", req.UserId),
+	// 	attribute.String("app.user.currency", req.UserCurrency),
+	// )
+	log.Infof("[PlaceOrder22] user_id=%q user_currency=%q", req.UserId, req.UserCurrency)
```

Comment:
why `Placeorder22` ?

Reply:
Resolved. Was added for testing if changes are being reflected.